src/components/Modal/Modal.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Button, Header, Icon, Modal } from 'semantic-ui-react'
+
+function ModalComponent(props) {
+
+  return (
+    <Modal
+  //    closeIcon
+      open={props.showModal}
+      onClose={() => props.closeModalPopup()}
+      onOpen={() => props.showModalPopup()}
+    >
+      <Modal.Content>
+        <p>
+         You are not logged in to place an order. Please continue to login
+        </p>
+      </Modal.Content>
+      <Modal.Actions>
+        <Button onClick={() => props.closeModalPopup()}>
+          <Icon name='remove' /> Cancel
+        </Button>
+        <Button color='blue' onClick={() => props.showModalPopup()}>
+          <Icon name='checkmark' /> Continue
+        </Button>
+      </Modal.Actions>
+    </Modal>
+  )
+}
+
+export default ModalComponent
```

Comment:
Here, instead of using Model multiple times, you can create One Model HOC and used commonly as reusable component. You have to pass only, open, onClose events from where it get calls.